block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go

```diff
@@ -169,11 +170,13 @@ var _ = framework.KubeDescribe("[Feature:Block_Volume_Attach_E2E]", func() {
 			if filestatus == true {
 				filepatharg2 := fmt.Sprintf("%s", portworxscpath)
 				cmd := exec.Command(pvscriptpath, filepatharg2, "portworxpvcreate")
-				//var stdout, stderr bytes.Buffer
-				//cmd.Stdout = &stdout
-				//cmd.Stderr = &stderr
+				var stdout, stderr bytes.Buffer
+				cmd.Stdout = &stdout
```

Comment:
Why are you setting empty stdout to cmd.Stdout?

Reply:
This got changed